test/teos/e2e/test_basic_e2e.py

```diff
@@ -129,6 +146,52 @@ def test_appointment_life_cycle(bitcoin_cli, create_txs):
     assert get_appointment_info(locator) is None
 
 
+def test_multiple_appointments_life_cycle(bitcoin_cli, create_five_txs):
```

Comment:
I'll rework this a bit when merging to the cli refactoring PR. I'll ping you to review.